subfun/getInfo.m

```diff
@@ -0,0 +1,70 @@
+function varargout = getInfo(BIDS, subID, info, session, run, type, opt)
```

Comment:
Was just checking this and wouldn't it make more sense to pass `opt` as argument number 4? Otherwise even when you query for Sessions you will have 5 arguments and this function won't return anything.

I might actually write a unit test for this: it is about time I try one of those.